README.md

```diff
@@ -23,7 +23,8 @@ the codebase can drastically change at any time. Also note that some parts of th
 'shiftable' yet, so modifying them could be difficult at this point.
 ```
 
-This is a WIP decompilation of The Legend of Zelda: Ocarina of Time.
+This is a WIP decompilation of The Legend of Zelda: Ocarina of Time. **It is not a PC port.** It has never been a PC port. There will never be a PC port in this repository. Do not try to claim that this project is a PC port. Anything claiming this is a PC port is misinformation.
```

Comment:
```suggestion
This is a WIP decompilation of The Legend of Zelda: Ocarina of Time. **It is not a PC port.** It has never been a PC port. There will never be a PC port in this repository. Please do not claim that this project is a PC port. Anything claiming this is a PC port is misinformed.
```
I think that sounds friendlier (but I wish we could set font size & color)
maybe add `You can get in touch with the team on our [Discord server](https://discord.zelda64.dev).` too

> You can get in touch with the team on our [Discord server](https://discord.zelda64.dev).

Reply:
I'd be happy with both of those changes. Anyone else want to comment before I change it?

Reply:
Well, we could change the size/colour using inline html, I think that works on GitHub.